src/QsCompiler/Transformations/Monomorphization.cs

```diff
@@ -210,14 +210,9 @@ private class NamespaceTransformation : NamespaceTransformation<TransformationSt
 
                 private bool NamespaceElementFilter(QsNamespaceElement elem)
                 {
-                    if (elem is QsNamespaceElement.QsCallable call)
-                    {
-                        return SharedState.ExemptCallableSet.Contains(call.Item.FullName);
-                    }
-                    else
-                    {
-                        return true;
-                    }
+                    return elem is QsNamespaceElement.QsCallable call
```

Comment:
nit: It would make it a bit easier for me to read if this were left as an if/else.

Reply:
done